docs/standard/base-types/character-encoding-introduction.md

```diff
@@ -281,7 +281,7 @@ A better approach is to break the string by counting grapheme clusters, or text
 
 :::code language="csharp" source="snippets/character-encoding-introduction/csharp/InsertNewlines.cs" id="SnippetGoodExample":::
 
-As noted earlier, however, in implementations of .NET other than .NET 5, the `StringInfo` class might handle some grapheme clusters incorrectly.
+As noted earlier, however, in implementations of .NET other than .NET 5+, the `StringInfo` class might handle some grapheme clusters incorrectly due to a bug in the `StringInfo` class.
```

Comment:
I find this a bit confusing. What about saying something like:

> Prior to .NET 5, the `StingInfo` class might handle some ....

Reply:
Good suggestion, thanks :)
Is the new commit clearer?